vendor/modules.txt

```diff
@@ -885,7 +885,7 @@ k8s.io/kube-openapi/pkg/spec3
 k8s.io/kube-openapi/pkg/util/proto
 k8s.io/kube-openapi/pkg/validation/spec
 # k8s.io/mount-utils v0.31.0-alpha.0
-## explicit; go 1.22
+## explicit; go 1.22.0
```

Comment:
Can do you do go mod tidy and go mod vendor to clean this up?

Reply:
run `go mod tidy` will change go.mod back to 1.22.0